security-jwt/src/main/java/io/micronaut/security/token/jwt/cookie/JwtCookieTokenReader.java

```diff
@@ -40,19 +43,33 @@
      */
     public static final Integer ORDER = 0;
 
+    @Deprecated
     protected final JwtCookieConfiguration jwtCookieConfiguration;
+    protected final AccessTokenCookieConfiguration accessTokenCookieConfiguration;
 
     /**
      *
      * @param jwtCookieConfiguration Configuration properties for JWT Cookie support
      */
+    @Deprecated
     public JwtCookieTokenReader(JwtCookieConfiguration jwtCookieConfiguration) {
         this.jwtCookieConfiguration = jwtCookieConfiguration;
+        this.accessTokenCookieConfiguration = jwtCookieConfiguration;
+    }
+
+    /**
+     *
+     * @param accessTokenCookieConfiguration Configuration properties for JWT Cookie support
+     */
+    @Inject
+    public JwtCookieTokenReader(AccessTokenCookieConfiguration accessTokenCookieConfiguration) {
+        this.jwtCookieConfiguration = null;
```

Comment:
should this be 

```java
this.jwtCookieConfiguration = accessTokenCookieConfiguration;
```

instead of 

```java
 this.jwtCookieConfiguration = null;
```